azure-mgmt-servicebus/src/main/java/com/microsoft/azure/management/servicebus/CheckNameAvailabilityResult.java

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.servicebus;
+
+import com.microsoft.azure.management.servicebus.implementation.CheckNameAvailabilityResultInner;
+
+/**
+ * The {@link com.microsoft.azure.management.servicebus.Namespaces#checkNameAvailability} action result.
+ */
+public class CheckNameAvailabilityResult {
+    private CheckNameAvailabilityResultInner inner;
+    /**
+     * Creates an instance of the check name availability result object.
+     *
+     * @param inner the inner object
+     */
+    public CheckNameAvailabilityResult(CheckNameAvailabilityResultInner inner) {
```

Comment:
The NamespaceImpl in the `implementation` namespace will be creating instance of this model. Thinking more about it, I think we can extract an interface out of this model and then it's impl can be in the `implementation` namespace. Will make the change.